package.json

```
@@ -95,7 +97,7 @@
     "uglify-js": "^3.1.3",
     "webpack": "^4.46.0",
     "yarnhook": "^0.5.1"
-  },
+	},
```

This looks like it has additional/unexpected whitespace.
```suggestion
  },
```